src/main/java/mate/academy/intro/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,23 @@
 package mate.academy.intro.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
use NotBlank for String validation
```suggestion
    @NotBlank
```